src/main/kotlin/com/hibob/academy/dao/PetDao.kt

```diff
@@ -0,0 +1,79 @@
+
+import com.hibob.academy.dao.*
+import jakarta.inject.Inject
+import org.jooq.DSLContext
+import org.jooq.Record
+import org.jooq.RecordMapper
+import org.springframework.stereotype.Component
+import java.sql.Date
+import kotlin.random.Random
+
+@Component
+
+class PetDao @Inject constructor(private val sql: DSLContext) {
+
+    private val pet = PetTable.instance
+    private val companyId = Random.nextLong()
```

Comment:
Why is the companyId a random long? we want to get explicitly the companyId as parameter in the functions